frontend/src/views/Transactions/_schema.js

```diff
@@ -88,7 +116,26 @@ export const transactionsColDefs = (t) => [
     field: 'updateDate',
     headerName: t('txn:txnColLabels.updateDate'),
     valueFormatter: dateFormatter,
-    width: 190
+    width: 190,
+    filter: 'agDateColumnFilter',
+    filterParams: {
+      filterOptions: ['inRange', 'equals', 'lessThan', 'greaterThan'],
+      defaultOption: 'inRange',
+      comparator: (filterDate, cellValue) => {
+          const cellDate = new Date(cellValue).setHours(0, 0, 0, 0);
```

Comment:
Not sure to understand, are you saying that transactions and compliance reports should handle in the same way `updatedDate`?

Reply:
Why is one like this: new Date(cellValue).setHours and the other just filterDate.setHours. It looks like you are modifying the passed in filterDate, so I was thinking it would also make sense to have new Date(filterDate).setHours()